pybars/tests/test__compiler.py

```diff
@@ -93,3 +93,8 @@ def test_segment_literal_notation(self):
         self.assertEqual(u"""hello""",
             render(u"""{{foo.[bar baz]}}""",
                 {'foo': {'bar baz': 'hello' }}))
+
+    def test_list_index_error(self):
+        self.assertEqual(u"""nothing: """,
```

Comment:
I see you're being consistent with the rest of the tests, but still wanted to point out that it's silly to use `"""` for a one-liner string. This would be much cleaner:
```
self.assertEqual(
    'nothing: ',
    render('nothing {{foo.1.bar}}', {'foo': []})
)
```

Reply:
Preserving existing code style 😬